pilz_testutils/CMakeLists.txt

```diff
@@ -34,7 +34,7 @@ if(CATKIN_ENABLE_CLANG_TIDY)
     message(FATAL_ERROR "clang-tidy not found.")
   else()
     message(STATUS "clang-tidy found: ${CLANG_TIDY_EXE}")
-    set(CMAKE_CXX_CLANG_TIDY "${CLANG_TIDY_EXE}")
+    set(CMAKE_CXX_CLANG_TIDY "${CLANG_TIDY_EXE}" "-header-filter=.*")
```

Comment:
Would it be equivalent to use `HeaderFilterRegex=".*"` in the `.clang-tidy` file?

Reply:
Should be. I will test it out and put it in the `.clang-tidy`, since that is probably preferred.

Reply:
@martiniil Done